src/formatDate.js

```diff
@@ -51,6 +51,34 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const newdate = date.split(`${fromFormat[fromFormat.length - 1]}`);
```

Comment:
you shood use 'camel case'
```suggestion
  const newDate = date.split(`${fromFormat[fromFormat.length - 1]}`);
```